src/config.rs

```diff
@@ -0,0 +1,65 @@
+extern crate yaml_rust;
+
+use std::collections::HashMap;
+use std::fs::File;
+use std::io::Read;
+use yaml_rust::yaml;
+
+pub struct Config {
+    pub readers: HashMap<String, Vec<String>>,
+}
+
+impl Config {
+    fn from_readers(readers: HashMap<String, Vec<String>>) -> Self {
+        Config { readers: readers }
+    }
+}
+
+pub fn get_config() -> Config {
+    let mut f = File::open("config.yaml").expect("Cannot open config.yaml");
```

Comment:
I would prefer the `.yml` file extension if you don't mind

Reply:
`.yaml` seems to be the recommended way: https://yaml.org/faq.html

Is it a no-go for you ?

Reply:
yes but nobody use this recommendation :stuck_out_tongue:

Reply:
We should try to stick to the standards in first place then adapt if there is any drawback